apps/passport-server/src/routing/routes/frogcryptoRoutes.ts

```diff
@@ -109,4 +114,47 @@ export function initFrogcryptoRoutes(
       )
     );
   });
+
+  app.get("/frogcrypto/user-state", async (req, res) => {
+    const result = await frogcryptoService.getUserState(
+      req.body as FrogCryptoUserStateRequest
+    );
+    res.json(result satisfies FrogCryptoUserStateResponseValue);
+  });
+
+  // TODO: serve real frog images
+  app.get("/frogcrypto/images/:uuid", async (req, res) => {
+    const imageId = checkUrlParam(req, "uuid");
+
+    let parsedUuid;
+    try {
+      parsedUuid = uuid.parse(imageId);
+    } catch (e) {
+      throw new PCDHTTPError(400, `invalid uuid ${imageId}`);
+    }
+
+    // convert to integer - see answers to https://stackoverflow.com/q/39346517/2860309
+    const buffer = Buffer.from(parsedUuid);
+    const result = buffer.readUInt32BE(0);
+
+    const frogPaths: string[] = [
+      "images/frogs/frog.jpeg",
+      "images/frogs/frog2.jpeg",
+      "images/frogs/frog3.jpeg",
+      "images/frogs/frog4.jpeg"
+    ];
+
+    res.redirect(
+      `${process.env.PASSPORT_CLIENT_URL}/${
+        frogPaths[result % frogPaths.length]
+      }`
```

Comment:
use `urlJoin` to make sure that the various `/` that show up in the url are correct.